src/impact-dashboard/CurveChart.tsx

```diff
@@ -137,6 +137,7 @@ const CurveChart: React.FC<CurveChartProps> = ({
   curveData,
   hospitalBeds,
   markColors,
+  // toggled,
```

Comment:
We should remove this before merging, unless we actually need it.

Reply:
Agree, will update.